casper/src/main/resources/NonNegativeNumber.rho

```diff
@@ -14,52 +14,71 @@
  9.  | 4,         | registry           | uri = rho:id:hxyadh1ffypra47ry9mk6b8r1i33ar1w9wjsez4khfe9huzrfcytx9
  ----+------------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------
  */
-new  NonNegativeNumber, MergeableTag, rs(`rho:registry:insertSigned:secp256k1`), uriOut in {
+new
+  NonNegativeNumber,
+  MergeableTag,
+  rs(`rho:registry:insertSigned:secp256k1`),
+  uriOut
+in {
   contract NonNegativeNumber(@init, return) = {
-    new this, valueStore in {
+    new
+      this,
+      valueStore
+    in {
+      // Adds `x` to the value if no overflow occurs, returns success message on success
       contract this(@"add", @x, success) = {
         if (x >= 0) {
-          for(@v <- @(*MergeableTag, *valueStore)){
+          for (@v <- @(*MergeableTag, *valueStore)){
```

Comment:
Conflicts in `NonNegativeNumber` is because  Issac's comments changes and the `MergeableTag`.